ImperatorToCk3Tests/CommonTests/VersionTests.cpp

```diff
@@ -4,44 +4,44 @@
 
 TEST(ImperatorWorld_VersionTests, VersionHigherOrEqual)
 {
-	const Version version1 = Version("1.4.1");
-	const Version version2 = Version("1.3.2");
-	const Version requiredVersion = Version("1.3.2.0");
+	const Version version1("1.4.1");
```

Comment:
You can drop version as it was moved into commonItems/GameVersion

Reply:
I wanted to do this in the next PR but can do now.